plotter.py

```diff
@@ -24,17 +26,14 @@ def plot_3D_points(xdata, ydata, zdata):
     plt.show()
 
 '''
-def plot_neutron_flux(3D_matrix_of_flux):
-
-
-plt.imshow(
-x = np.linspace(-1, 1, 100)
-y = np.cos(x)
-
-plt.plot(x, y, 'k.-')
-plt.xlabel('x')
-plt.ylabel('y')
-plt.title('Simple Matplotlib Example')
-plt.show()
+ @brief     Creates a heat map of the flux data
+ @param     flux_data a 3d numpy array containing the flux information
+ @param     index the z cell-value at which the heat map is displayed
 '''
+def plot_heat_map(flux_data, index):
 
+    img = flux_data
+    lum_img = img[:, :, index]
```

Comment:
Well first off, we shouldn't be running problems that fine yet. But let's do some simple math. If we have 0.05 spacing that means nx is 40 on a mesh from -1 to 1. With ``repeat = 100`` that means we would have ``plot_array`` with dimensions 4000 x 4000, meaning 16 million elements.  Double precision numbers are 8 bytes so that equates to 128 MB. That's a lot but I think you're memory should still be able to handle it.

But probably a better idea is to have ``repeat`` be allowed to vary, for instance have it be an argument to the function or come up with some heuristic for a good number depending on the size of the mesh, etc.